layouts/specifications/list.html

```diff
@@ -31,7 +31,11 @@ <h1>
       {{ range .Page.Sections.ByWeight}}
         <li>
           <h3><a href="{{- with .Params.redirect_url | default .RelPermalink }} {{.}} {{- end }}">{{ .Title }}</a></h3>
-          <p>{{ .Params.summary | truncate 57 }}</p>
+          {{ if eq .Params.summary60Char nil  }}
```

Comment:
you can use `if isset .Params "summary60Char"` though I'm not sold on the name of this parameter, as it breaks the convention we currently follow for parameter names.

Reply:
`if isset .Params "summary60Char"` doesn't work. Believe me, I tried... The problem is that the parameter does not need to be there, thus undefined (or nil).
Regarding the name, that's what was agreed upon when this was initially implemented (and somehow disappeared in a redesign of the page).